src/components/SearchBar.js

```diff
@@ -42,10 +45,25 @@ class SearchBar extends Component {
 
     render() {
         return (
-            <form onSubmit={this.handleSubmit}>
-                <input type="text" value={this.state.value} onChange={this.handleChange} />
-                <input type="submit" value="Submit" />
-            </form>
+            <div>
+                <MuiThemeProvider>
```

Comment:
I read that documentation as well, however unless we render the MuiThemeProvider on the server, an error is always going to appear when we try to add a material-ui component anywhere in the codebase without the MTP - at least from my experience. Hence, we need to either painstakingly sprinkle the MTP all over the place, as I have been, or we need to render it once on the server by feeding it to a middleware such as an autoprefixer (we can easily accomplish this w/ Node.js).